jest.config.js

```diff
@@ -29,6 +29,6 @@ module.exports = {
   testPathIgnorePatterns: ["<rootDir>/packages/(?:.+?)/(esm|lib)/"],
   transform: {
     "\\.[jt]sx?$": "babel-jest",
-    "\\.ya?ml$": "jest-yaml-transform"
+    "\\.ya?ml$": "jest-transform-yaml"
   }
```

Comment:
The old library had not been updated in a long time and did not handle arrays correctly. Arrays of objects got flattened with a number and a dot in the property names instead of being an actual array. The new library correctly handles this.